### PR TITLE
Implement recent issues

### PR DESCRIPTION
**Pull request by mrElnekave**
Implemented issue #165 

@tinmarr plz check line 48 sound.py for correctness.

        active_channels (Dict[int, Sound]): A dictionary housing all the active sounds, stored by their name.

### CODE REVIEW

File: rubato/game.py [hunk 1]
```diff
@@ -97,6 +97,8 @@ def constant_loop(cls):  # test: skip
         try:
             while True:
                 cls.update()
+        except KeyboardInterrupt:
+            sys.exit(0)
```
Comment (tinmarr) [inline rubato/game.py:101]: Should also quit SDL. Use the quit code that's in the SDL quit event handling

**Review by tinmarr — CHANGES_REQUESTED** [review]
Just one small change. (see comment)

File: rubato/game.py (hunk 1 above)
Comment (mrElnekave) [reply rubato/game.py:101]: Cool beans

**Review by tinmarr — APPROVED** [review]
Looks good

**Pull request merged**